process/sp.py

```diff
@@ -16,10 +16,13 @@
 import numpy as np
 import os
 import setup_sp as ssp
+from setup_config import * # import project config parameters
```

Comment:
See earlier namespace management comments about just importing the module rather than importing * from it.